upcoming/composable-pipelines-pattern.md

```diff
@@ -0,0 +1,392 @@
++++
+Author = ["Samuel Lampa"]
+date = "2015-01-30T00:00:00-08:00"
+series = [""]
+title = "Pattern for pluggable pipeline components in Go"
++++
+
+# Patterns for composable concurrent pipelines in Go
```

Comment:
Hi Nathan,

Is there anything more I need to do about the blog post?

(I recently needed to link to the post, so were a bit eager to get it 
posted, in case you want to include it :) (Otherwise no worries, then 
I'll post it elsewhere))

Cheers
// Samuel


On 2015-02-07 20:39, Nathan Youngman wrote:
>
> In upcoming/composable-pipelines-pattern.md 
> <https://github.com/gopheracademy/gopheracademy-web/pull/114#discussion_r24292092>:
>
> > @@ -0,0 +1,392 @@
> > ++++
> > +Author = ["Samuel Lampa"]
> > +date = "2015-01-30T00:00:00-08:00"
> > +series = [""]
> > +title = "Pattern for pluggable pipeline components in Go"
> > ++++
> > +
> > +# Patterns for composable concurrent pipelines in Go
>
> You're title here and above don't match.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/gopheracademy/gopheracademy-web/pull/114/files#r24292092>.
>


-- 
Samuel Lampa
-----------------------------------------------
Systems Developer at BILS (bils.se)
PhD student at Uppsala University (farmbio.uu.se)
-----------------------------------------------
    Mail: samuel.lampa@bils.se <- NEW! Please update!
  Google: samuel.lampa@gmail.com
   Skype: samuellampa
  Mobile: +46 70-207 3732
    Blog: http://samuel.lampa.co
Twitter: http://twitter.com/smllmp
-----------------------------------------------